src/containers/Storage/Vdisk/__tests__/colors.tsx

```diff
@@ -63,26 +63,39 @@ describe('VDisk state', () => {
             <>
                 <VDisk VDiskId={{Domain: 1}} />
                 <VDisk VDiskId={{Domain: 2}} VDiskState="OK" />
-                <VDisk VDiskId={{Domain: 3}}                 DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 4}}                                   FrontQueues="Green" />
+                <VDisk VDiskId={{Domain: 3}} DiskSpace="Green" />
+                <VDisk VDiskId={{Domain: 4}} FrontQueues="Green" />
                 <VDisk VDiskId={{Domain: 5}} VDiskState="OK" DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 6}} VDiskState="OK"                   FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 7}}                 DiskSpace="Green" FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 8}} VDiskState="OK" DiskSpace="Green" FrontQueues="Green" />
```

Comment:
those lines were specifically aligned to see the pattern of specified props
it does not match any linter rule, but it was intentional